docs/element-templates/README.md

```diff
@@ -309,78 +329,167 @@ Notice that adherence to the following configuration options is enforced by desi
 | **Binding parameters**  | `name`: the name of the property  |
 | **Mapping result** | `<... [name]=[userInput] ... />`  |
 
-##### `camunda:property`
+The `property` binding is supported both in Camunda Platform and Cloud.
```

Comment:
Also, for this binding I would add a heading (as below): `Bindings working for both, Camunda Platform and Camunda Cloud`

Reply:
Maybe `Product Agnostic Bindings`?

Reply:
I like to have the explicit support in the headings. What about

```
Bindings for `Camunda Platform` or `Camunda Cloud`
Bindings for `Camunda Platform`
Bindings for `Camunda Cloud`
```